### PR TITLE
Remove remaining Foundation typography classes

### PR DESCRIPTION
**Pull request by ryan-ludwig**
VUE-849

### CODE REVIEW

File: src/assets/scss/global/typography.scss [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
 
-h1.huge {
-	@include huge-headline();
-}
-
-.small-text {
-	@include small-text();
-}
-
 .featured-text {
```
Comment (ryan-ludwig) [inline src/assets/scss/global/typography.scss:3]: These remaining 3 classes are still in use on Homepage[Contentful] components. I'm not sure which are still in play. Probably best to wait for CORE-51 to remove.

Comment (mcstover) [reply src/assets/scss/global/typography.scss:3]: Oh interesting...

**Review by mcstover — APPROVED** [review]
Would like to hold for merge until after we deploy the promo bar hotfix

File: src/assets/scss/global/typography.scss (hunk 1 above)
Comment (eddieferrer) [reply src/assets/scss/global/typography.scss:3]: @ryan-ludwig can you make a note of this in CORE-51 so we don't forget to remove these classes when that work is done

**Review by eddieferrer — APPROVED** [review]

**Pull request merged**